src/config/context/model.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-console */
+import { injectable } from 'tsyringe';
+import fs from 'fs';
+import { CONTEXTFILE_PATH } from '../../constants';
+
+export interface IContext {
+  readonly current?: string,
+  readonly store: {
+    [name: string]: string
+  }
+}
+
+export class Context implements IContext {
+  current?: string | undefined;
+  store: { [name: string]: string; };
+
+  constructor(ctx: IContext) {
+    this.current = ctx.current;
+    this.store = ctx.store;
+  }
+
+  getContext(contextName: string) {
+    return this.store[contextName as string];
+  }
+}
+
+export interface IContextAllocator {
+  contextFilePath?: string
+  load: () => Context | undefined
+  save: (context: Context) => Context | undefined
+}
+
+@injectable()
+export class ContextAllocator implements IContextAllocator {
+  contextFilePath = CONTEXTFILE_PATH;
+  load(): Context | undefined {
+    try {
+      return new Context(JSON.parse(fs.readFileSync(this.contextFilePath, 'utf8')) as IContext);
+    } catch (error) {
+      return undefined;
+    }
+  }
+
+  save(context: Context) {
+    try {
+      fs.writeFileSync(this.contextFilePath, JSON.stringify({
+        current: context.current,
+        store: context.store
+      }), { encoding: 'utf8' });
+      return context;
+    } catch (error) {
+      console.warn(error);
```

Comment:
You can disable only one line:

```suggestion
      console.warn(error); // eslint-disable-line no-undef, no-console
```

and then please remove 

```
/* eslint-disable no-undef */
/* eslint-disable no-console */
```

from the beginning of file :)

Reply:
Please also add: 

```
console.warn(error);
```

in the `load` function :)

Reply:
This still gives error 
![image](https://user-images.githubusercontent.com/41781438/134152018-a289a49f-47f3-413f-b709-80972ab6ed34.png)

Reply:
You have to write `eslint-disable-line` in this same line:

```js
console.warn(error); // eslint-disable-line no-undef, no-console
```

Reply:
Yeah, my bad just figured that.